src/applications/gi/containers/Modals.jsx

```diff
@@ -292,6 +292,10 @@ export class Modals extends React.Component {
           <p>When you select some schools, we import the average tuition/fees for an undergraduate student as reported by the school to the Department of Education through <a href="http://nces.ed.gov/ipeds/datacenter/" id="anch_442" target="blank">IPEDS</a>. This is the same information that is published on <a href="http://nces.ed.gov/collegenavigator/" id="anch_443" target="blank">College Navigator</a>.</p>
           <p>To learn more, please review our "<a href={'http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#yellow_ribbon_from_school'} target="_blank">About This Tool</a>" page.</p>
         </Modal>
+        <Modal onClose={this.props.hideModal} visible={this.shouldDisplayModal('calcYellowRibbon')}>
+          <h3>Yellow ribbon</h3>
+          <p>This is a placeholder</p>
```

Comment:
This [needs](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9365#issuecomment-397125131) content.